### DISCUSSION TITLE
DisplacementMap for StandardMaterial

### DISCUSSION THREAD

**Discussion by goldenratio** [jnsmalm/pixi3d#88] [Q&A] [answered]
Hi,

was wondering do we have DisplacementMap like in three.js https://threejs.org/docs/#api/en/materials/MeshStandardMaterial.displacementMap in StandardMaterial?

**Comment by jnsmalm** [comment, marked as answer]
This is not supported when using standard material. You have to write your own material if this is needed, you can find example how to do this at https://codesandbox.io/s/github/jnsmalm/pixi3d-sandbox/tree/master/custom-material

**Reply by goldenratio** [reply]
I will take a look. My knowledge on this is bit limited. Thanks for the help.
